<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>파일(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="new_window"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_U"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>편집(&amp;E)</string>
    </property>
    <addaction name="action_U_2"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
    <addaction name="action_P_2"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="actionBing_S"/>
    <addaction name="action_F"/>
    <addaction name="action_N_2"/>
    <addaction name="action_V"/>
    <addaction name="action_R"/>
    <addaction name="action_G"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>서식(&amp;O)</string>
    </property>
    <addaction name="action_W_2"/>
    <addaction name="action_F_2"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>보기&amp;(V)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>확대하기/축소하기</string>
     </property>
     <addaction name="action_I"/>
     <addaction name="action_O_2"/>
     <addaction name="action_8"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="action_S_checkable"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>도움말(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="action_F_3"/>
    <addaction name="separator"/>
    <addaction name="action_A_3"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_N">
   <property name="text">
    <string>새로 만들기(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="new_window">
   <property name="text">
    <string>새 창(&amp;W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>열기(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>저장(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>다른 이름으로 저장(&amp;A)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>페이지 설정(&amp;U)...</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>인쇄(&amp;P)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>끝내기(&amp;X)</string>
   </property>
  </action>
  <action name="action_U_2">
   <property name="text">
    <string>실행 취소(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>잘라내기(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>복사(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_P_2">
   <property name="text">
    <string>붙여넣기(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>삭제(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBing_S">
   <property name="text">
    <string>Bing으로 검색(&amp;S)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>찾기(&amp;F)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_N_2">
   <property name="text">
    <string>다음 찾기(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>이전 찾기(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>바꾸기(&amp;R)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>이동(&amp;G)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>모두 선택(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>시간/날짜(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_W_2">
   <property name="text">
    <string>자동 줄 바꿈(&amp;W)</string>
   </property>
  </action>
  <action name="action_F_2">
   <property name="text">
    <string>글꼴(&amp;F)...</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>확대(&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_O_2">
   <property name="text">
    <string>축소(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>확대하기/축소하기 기본값 복원</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_S_checkable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>상태 표시줄(&amp;S)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>도움말 보기(&amp;H)</string>
   </property>
  </action>
  <action name="action_F_3">
   <property name="text">
    <string>피드백 보내기(&amp;F)</string>
   </property>
  </action>
  <action name="action_A_3">
   <property name="text">
    <string>메모장 정보(&amp;A)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
